core/src/main/resources/lib/form/expandableTextbox.jelly

```diff
@@ -78,8 +78,8 @@ THE SOFTWARE.
             </j:if>
           </div>
           <div class="expanding-input__button">
-            <input type="button" value="&#x25BC;" onclick="expandTextArea(this,'textarea.${h.jsStringEscape(name)}')"
-                   tooltip="${%tooltip}"/>
+            <button type="button" class="jenkins-button expandable-textbox-button jenkins-!-margin-left-1" onclick="expandTextArea(this,'textarea.${h.jsStringEscape(name)}')"
```

Comment:
does the `expandable-textbox-button` class do anything? I couldn't see any reference to it